test/e2e/storage/persistent_volumes-local.go

```diff
@@ -252,6 +252,39 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		})
 	})
 
+	Context("when pod using local volume with non-existant path", func() {
+		It("should not be able to mount", func() {
+			for _, testVolType := range LocalVolumeTypes {
+				By(fmt.Sprintf("local-volume-type: %s", testVolType))
+				testVol := &localTestVolume{
+					node:            config.node0,
+					hostDir:         "/non-existent/location/nowhere",
+					localVolumeType: testVolType,
+				}
+				By("Creating local PVC and PV")
+				createLocalPVCPV(config, testVol)
+				_, pod1Err := createLocalPod(config, testVol)
+				Expect(pod1Err).To(HaveOccurred())
```

Comment:
Can you also check the Events on that Pod, and look for some failed mount because path doesn't exist message?

Reply:
From the errors generated I could only see the timeout message for starting the pod. I am not sure if there is a visibility for specific error message, but I will double check.

Reply:
I think you can do a Pod.Get, and look at its Events.